File: QTD.SPA/src/app/_DtoModels/RR_EnablingObjective/RR_EO_LinkOptions.ts
import { EnablingObjective } from "../EnablingObjective/EnablingObjective";
import { RegulatoryRequirement } from "../RegulatoryRequirements/RegulatoryRequirement";

export class RR_EO_LinkOptions{
  regulatoryRequirementId!: any;
  EOIds!: any[];
  changeNotes!:string;
  effectiveDate!:Date;
  regulatoryRequirement!: RegulatoryRequirement;
  enablingObjective!: EnablingObjective;
}

File: QTD2.Infrastructure/Model/RR_EO_Link/RR_EO_LinkOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.RR_EO_Link
{
    public class RR_EO_LinkOptions
    {
        public int RegulatoryRequirementId { get; set; }

        public int[] EOIds { get; set; }

        public string ChangeNotes { get; set; }

        public DateTime EffectiveDate { get; set; }
    }
}
